src/main/webapp/app/entities/club-family/family-member-create.tsx

```diff
@@ -0,0 +1,145 @@
+import React, { ReactNode } from 'react';
+
+import { Link, RouteComponentProps } from 'react-router-dom';
+import { Button, Row, Col, Label } from 'reactstrap';
+import { AvForm, AvGroup, AvInput, AvField } from 'availity-reactstrap-validation';
+import { Translate, translate } from 'react-jhipster';
+import { connect } from 'react-redux';
+import { getUsersWithoutFamily } from 'app/modules/administration/user-management/user-management.reducer';
+import { createEntity } from 'app/entities/user-cc-info/user-cc-info.reducer';
+import { IRootState } from 'app/shared/reducers';
+import { IUser } from 'app/shared/model/user.model';
+
+export interface IFamilyMemberCreateProps extends StateProps, DispatchProps, RouteComponentProps<{ id: string }> {}
+
+class FamilyMemberCreate extends React.Component<IFamilyMemberCreateProps> {
+  constructor(props: IFamilyMemberCreateProps) {
+    super(props);
+  }
+
+  componentWillUpdate(nextProps: IFamilyMemberCreateProps) {
+    if (nextProps.updateSuccess !== this.props.updateSuccess && nextProps.updateSuccess) {
+      this.handleClose();
+    }
+  }
+
+  handleClose = () => {
+    this.props.history.push(`/entity/members/club-family/${this.props.match.params.id}`);
+  };
+
+  componentDidMount() {
+    this.props.getUsersWithoutFamily();
+  }
+
+  renderNames(users: readonly IUser[]): ReactNode {
+    return users.map((user: IUser) => {
+      if (user) {
+        return <option key={user.id} value={user.id}>{`${user.firstName ?? ''} ${user.lastName ?? ''}`}</option>;
```

Comment:
Can use concatFullName in string-utils

Reply:
Resolve in dfc7a4da3bb5aa771571d1c94115bf52c2f87f19